src/pages/PlayPage.spec.js

```diff
@@ -1,17 +1,56 @@
 import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 import PlayPage from './PlayPage'
 
-describe('PlayPage', () => {
-  it('has 4 Button component rendered', () => {
-    const answers = [
-      { title: 'Bulls on Parade', right: true, id: '1' },
-      { title: 'Bomb Track', wrong: true, id: '2' },
-      { title: 'People of the Sun', wrong: true, id: '3' },
-    ]
+const answers = [
+  { title: 'Bulls on Parade', right: true, id: '1' },
+  { title: 'Bomb Track', wrong: true, id: '2' },
+  { title: 'People of the Sun', wrong: true, id: '3' },
```

Comment:
why do you have right and wrong? could you not just set the right to false?

Reply:
That's because there three different options. The right button is green the wrong button is red and the default is grey. But i will go and think about it.